client/client.go

```diff
@@ -376,18 +384,7 @@ func (c *CvpClient) makeRequest(reqType string, url string, params *url.Values,
 		if err = json.Unmarshal(resp.Body(), &info); err == nil {
 			// check and see if we have a CVP error payload
 			if err = info.Error(); err != nil {
-				// Unauthorized User
-				if info.ErrorCode == "112498" {
-					// retry to same node/host
```

Comment:
Is this section being removed because at this point develop is intended to track updates to the v2 branch?

Reply:
yes.